### ISSUE TITLE
Swift Package Manager

### ISSUE THREAD

**Issue by Shakarang** [zenangst/Hue#38]
Hello,

Will you add the support of the Swift Package Manager?

**Comment by zenangst** [comment]
Yeah I think we will do that eventually but it is not really something that we will be focusing on ourselves as we use `Carthage` and/or `Cocoapods` at Hyper. But feel free to make a PR 😎

**Comment by ghecho** [comment]
I would also want this, however, It's not currently possible since the Swift Package Manager only compiles for host platforms, this means that only macOS and Linux are supported.

I just stumbled upon an issue with Carthage, it all goes well until you submit the app to the App Store, it throws the following errors when you are uploading the archive to the store via Xcode:

ITMS-90685 CFBundleIdentifier Collision:
Cache
Hue

ITMS-90205 Invalid Bundle:
Imaginary
Lightbox

ITMS-90206 Invalid Bundle:
Imaginary
Lightbox

My Cartfile only contains this:
github "hyperoslo/Lightbox"
github "hyperoslo/Imaginary"

Apparently I'm not the only one with this issue, as far as I can understand, this person is also having the same issue https://github.com/hyperoslo/Lightbox/issues/109 

IMPORTANT: This is not an issue with Hue as far as I know because if you only use Hue it al goes well in the submission, the error arises when you include Lightbox (which has a dependency on Hue) and then it throws those errors.

I tried to add support to SPM to Hue, Lightbox, Imaginary, etc. because of that issue but after some time I realized that it was not possible at the moment due to the host-platform-only issue.
